dolphinscheduler-ui/src/views/projects/workflow/components/dag/dag-context-menu.tsx

```diff
@@ -126,7 +126,7 @@ export default defineComponent({
     const handleCopy = () => {
       const genNums = 1
       const type = props.cell?.data.taskType
-      const taskName = uuid(props.cell?.data.taskName + '_')
+      const taskName = uuid(props.cell?.data.taskName + '-')
```

Comment:
Modifying the global rule directly will cause other task types to be abnormal, so I think we should create a new rule for the k8s task type.